src/platform/intel/cavs/platform.c

```diff
@@ -389,26 +389,6 @@ int platform_init(struct sof *sof)
 	trace_point(TRACE_BOOT_PLATFORM_CPU_FREQ);
 
 #if CONFIG_TIGERLAKE
```

Comment:
shall we remove `CONFIG_CAVS_LPRO_ONLY` then too - in a separate commit?

Reply:
updated, thanks!